modules/payloads/singles/windows/x64/download_exec.rb

```diff
@@ -0,0 +1,211 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+module MetasploitModule
+
+    CachedSize = 443
+  
+    include Msf::Payload::Windows
+    include Msf::Payload::Single
+    include Msf::Payload::Windows::BlockApi
```

Comment:
This is going to use the 32-bit BlockApi when we should be using the 64-bit one.

```suggestion
    include Msf::Payload::Windows::BlockApi_x64
```